simple_test.sh

```diff
@@ -1,48 +1,81 @@
 #!/bin/sh
 
+# This test script automates our distributed work system. When running this script, please run it as follows:
+# ./simple_test [hostname] [# of workers] [test filename]
+#
+# [hostname] refers to the complete address with "http://" or "https://" preceding it.
+# [test filename] refers to the test file located in the test directory.
+
+
 PROJECT_DIRECTORY=$(pwd)
 
+# Command-line args
+hostname=$1
+worker_cnt=$2
+test_file=$3
```

Comment:
I think we should have some more sophisticated command line arguments.

See [this little script for an example](https://github.com/jmunixusers/presentations/blob/main/CommandLine.md#grand-finale)